tests/README.md

```diff
@@ -114,6 +126,7 @@ Run using
 ```
 (.venv3)$ python3 tests/stubtest_third_party.py
 ```
+*Note: Due to [a bug in venv](https://github.com/python/cpython/issues/87915), on Windows you **have to** use `python` as your executable. `python3` will **not** work.*
```

Comment:
I'm not sure this note belongs here: it seems to apply to all our tests:

```pycon
(.venv) C:\Users\alexw\coding\typeshed>python3 tests/check_new_syntax.py
Python was not found; run without arguments to install from the Microsoft Store, or disable this shortcut from Settings > Manage App Execution Aliases.
```

Can we take out this note for now, and maybe work on improving the docs for those on Windows in (yet) another PR?

Reply:
Sure. Btw just FYI, you don't have to install it from the store (which I've had very bad experience with as well). You can disable that store link by going under settings > App execution aliases, and disabling it
![image](https://user-images.githubusercontent.com/1350584/193471514-04050533-ca29-4da2-b1a8-ee3134b315ac.png)

Reply:
I've got used to typing "python" (which is one fewer character anyway) whenever instructions on how to run a script say "python3", so it doesn't matter too much to me :)